src/Platform/src/Platform.Windows.cpp

```diff
@@ -231,6 +232,47 @@ namespace OpenLoco::Platform
 
         return true;
     }
+
+    std::vector<std::string> getCmdLineVector(int, const char**)
+    {
+        // Discard the input as it will either be ASCII or not available
+
+        // We need to get the Wide command line and convert it to utf8 command line parameters
+        const auto cmdline = GetCommandLineW();
+
+        int argc{};
+        auto* argw = CommandLineToArgvW(cmdline, &argc);
+        if (argw == nullptr)
+        {
+            std::cerr << "CommandLineToArgvW failed to convert commandline";
+            // We will continue but just ignore the command line args.
+            argc = 0;
+        }
+
+        // This will hold a utf8 string of the command line
+        std::vector<std::string> argvStrs;
+        argvStrs.resize(argc);
+
+        for (auto i = 0; i < argc; ++i)
+        {
+            int length = WideCharToMultiByte(CP_UTF8, 0, argw[i], -1, 0, 0, NULL, NULL);
+
+            if (length == 0)
+            {
+                // Sadly can't print what the argument is that is causing the issue as it needs to be in utf8...
```

Comment:
perhaps print the raw bytes? might be better than nothing

Reply:
hmm not sure its too wise as pretty sure iostreams would be using this function to print anyway. I can't really ever see this fail tbh.